tests/unit/test_acoustic_sync.py

```diff
@@ -0,0 +1,128 @@
+import datetime
+from unittest import mock
+from unittest.mock import MagicMock
+
+import pytest
+
+from ctms import acoustic_service
+
+CTMS_ACOUSTIC_CLIENT_ID = "CLIENT"
+CTMS_ACOUSTIC_CLIENT_SECRET = "SECRET"
+CTMS_ACOUSTIC_REFRESH_TOKEN = "REFRESH"
+CTMS_ACOUSTIC_MAIN_TABLE_ID = "1"
+CTMS_ACOUSTIC_NEWSLETTER_TABLE_ID = "9"
+
+
+@pytest.fixture()
+def no_acoustic():
+    patcher = mock.patch("ctms.acoustic_service.Acoustic")
+    patcher.start()
+    yield patcher
+    patcher.stop()
```

Comment:
I think the purpose is to mock ``Acoustic`` so it doesn't call anything, but it looks like the mock is unused in the code, so it doesn't really need to be returned. It could be moved to ``base_ctms_acoustic_service``,  something like:

```python
def base_ctms_acoustic_service():
    with mock.patch("ctms.acoustic_service.Acoustic"):
        yield acoustic_service.CTMSToAcousticService(
            client_id=CTMS_ACOUSTIC_CLIENT_ID,
            client_secret=CTMS_ACOUSTIC_CLIENT_SECRET,
            refresh_token=CTMS_ACOUSTIC_REFRESH_TOKEN,
            acoustic_main_table_id=CTMS_ACOUSTIC_MAIN_TABLE_ID,
            acoustic_newsletter_table_id=CTMS_ACOUSTIC_NEWSLETTER_TABLE_ID,
            server_number=6,
        )
```